python/lsst/db/engineFactory.py

```diff
@@ -25,9 +25,11 @@
 
 @author  Jacek Becla, SLAC
 """
+from future import standard_library
+standard_library.install_aliases()
 
 # standard library imports
-from ConfigParser import ConfigParser, NoSectionError
+from configparser import ConfigParser, NoSectionError
```

Comment:
I see that future versions of future are dropping `configparser` and recommending that people install the PyPI compatibility version on python 2. The alternative is to try/except:
```python
try:
   from ConfigParser import ...
except ImportError:
   from configparser import ...
```